RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -32,6 +32,27 @@
     nSigmaZ = 5.0
 )))
 
+from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
+from RecoTracker.TkTrackingRegions.globalTrackingRegionWithVertices_cfi import globalTrackingRegionWithVertices as _globalTrackingRegionWithVertices
+
+pp_on_XeXe_2017.toReplaceWith(detachedQuadStepTrackingRegions,
+                              _globalTrackingRegionWithVertices.clone(RegionPSet=dict(
+            precise = True,
+            useMultipleScattering = False,
+            useFakeVertices       = False,
+            beamSpot = "offlineBeamSpot",
+            useFixedError = True,
+            nSigmaZ = 4.0,
+            sigmaZVertex = 4.0,
+            fixedError = 3.75,
+            VertexCollection = "firstStepPrimaryVertices",
+            ptMin = 0.8,
+            useFoundVertices = True,
+            originRadius = 1.5
+            ))
+                              )
```

Comment:
This block is copy-pasted along the concerned files. Would it be possible to define the customization once and then use that definition everywhere?